### PR TITLE
Lm_Kiviads Bid Adapter : initial adapter release

### PR DESCRIPTION
**Pull request by KiviAds**
## Type of change
- [x] New bidder adapter

## Description of change
New prebid adapter to connect to kiviads.com demand source

- contact email of the adapter’s maintainer: pavlo@xe.works
- test parameters for validating bids:
```
[
    {
        code: 'test-banner',
        mediaTypes: {
            banner: {
                sizes: [[300, 250]],
            }
        },
        bids: [
            {
                bidder: 'lm_kiviads',
                params: {
                    env: 'lm_kiviads',
                    pid: '40',
                    ext: {}
                }
            }
        ]
    },
    {
        code: 'test-video',
        sizes: [ [ 640, 480 ] ],
        mediaTypes: {
            video: {
                playerSize: [640, 480],
                context: 'instream',
                skipppable: true
            }
        },
        bids: [{
            bidder: 'lm_kiviads',
            params: {
                env: 'lm_kiviads',
                pid: '40',
                ext: {}
            }
        }]
    }
];
```

## Other information
Docs PR: https://github.com/prebid/prebid.github.io/pull/4829

### CODE REVIEW

**Comment by bretg** [comment]
@KiviAds - would it be possible to change your `placement` param to `pid` to make it match your server-side parameter?

**Comment by KiviAds** [comment]
change placement -> pid according to server-side parameters

**Comment by KiviAds** [comment]
@lksharma Hi, could you please check this pr? Thank you

**Review by lksharma — DISMISSED** [review]
LGTM

**Review by lksharma — CHANGES_REQUESTED** [review]

File: modules/lm_kiviadsBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  if (typeof floor === 'object' && !isNaN(floor.floor) && floor.currency === CUR) {
+    return floor.floor;
+  }
+
+  return null;
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['kivi', 'kiviads'],
```
[the remark this answers is not in the document]
Comment (KiviAds) [reply modules/lm_kiviadsBidAdapter.js]: @lksharma, thanks for the information. The mistake has been fixed

**Review by lksharma — APPROVED** [review]
LGTM

**Pull request merged**